src/test/java/com/flightstats/hub/cluster/CuratorClusterTest.java

```diff
@@ -34,17 +41,19 @@ void testPath() throws Exception {
                 "/SpokeCluster",
                 false,
                 true,
-                new SpokeDecommissionCluster(curator, new SpokeProperties(PropertiesLoader.getInstance())),
+                new SpokeDecommissionCluster(curator, spokeProperties, localHostProperties),
                 new AppProperties(PropertiesLoader.getInstance()),
```

Comment:
Would it make sense to use the variables that are newly defined above here for `appProperties` and `spokeProperties`?